src/20_Components/amp-addthis.html

```diff
@@ -0,0 +1,84 @@
+<!---{
+  "disablePlayground": true
```

Comment:
Is there a particular reason why you've disabled the playground? I usually prefer to have it enabled by default as it's a great way to further explore a component.

Reply:
I disabled playground because I wasn't working locally and didn't want to push it in that state. That could just be a local configuration error, so I can remove this if it should work out of the box.